HISTORY.md

```diff
@@ -1,6 +1,31 @@
 Release History
 ---------------
 
+## 0.8.0 (2024-08-15)
+
+### Special Notes
+
+- Remove Tornado visualization server and replace it with Solara that is compatible with Jupiter notebooks (https://github.com/projectmesa/mesa-geo/pull/212). This is in line with Mesa's recent changes to use Solara for visualization.
```

Comment:
I would add specifically that 0.8 is the last release series compatible with Mesa 2.3.x, and that Mesa-geo 1.0 will be compatible with Mesa 3.0.

Reply:
Updated accordingly

Reply:
Thanks!